src/main.rs

```diff
@@ -57,7 +59,14 @@ fn main() -> std::io::Result<()> {
         None
     };
 
-    let toc = TableOfContent::new(&settings.storage, runtime, consensus_enabled);
+    let first_peer = if settings.cluster.enabled {
```

Comment:
I am a bit confused here :)

The code reads as: the cluster feature is off therefore it is a `first_peer`.

I thought that the concept of `first_peer` would only make sense when the cluster feature is enabled.
Otherwise it is simply a single node deployment.

I was expecting to find `first_peer` on the `ConsensusEnabled` struct for instance and feed it to the `toc`,

Reply:
Yeah, this is confusing a bit, let's have a quick call to discuss it

Reply:
Ok, I think I came up with a solution that doesn't leak details to main.